### PR TITLE
build(ui): Enable css sourcemaps

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -268,7 +274,8 @@ var legacyCssConfig = {
         loader: 'file-loader?name=' + '[name].[ext]'
       }
     ]
-  }
+  },
+  devtool: '#source-map'
```
Comment (billyvg) [inline webpack.config.js]: For JS, we use cheap source maps in dev for performance reasons, I think it's fine to always use `source-map` for css if the performance is negligible

Comment (lynnagara) [reply webpack.config.js]: I wasn't even able to get `cheap-source-map` to work here

**Comment by lynnagara** [comment]
Updated this to be set via an environment variable - but off by default - as it negatively affects build times

**Comment by MaxBittker** [comment]
how do i verify this locally? i ran `WITH_SOURCEMAPS=1 sentry devserver` and they seemed to build but i'm still setting sentry.css in the developer tools

**Comment by lynnagara** [comment]
@MaxBittker That's exactly what I'm doing 🤔. Could you try maybe clearing browser cache, nuking `dist` or triggering a change to a `.less` file?

**Comment by MaxBittker** [comment]
ah, that seemed to do the trick! it's showing me uniminified css though, not .less, is that what yours does? it's still helpful

**Comment by MaxBittker** [comment]
oops, i was wrong again, it's totally working all the way to less, awesome

**Review by MaxBittker — APPROVED** [review]
possibly not needed, but what do you think of changing the name to STYLE_SOURCEMAPS or something that indicates this is css specific? either way lgtm, nice improvement 👍

**Comment by lynnagara** [comment]
@MaxBittker That makes sense, but I don't have a strong opinion about it so I'm just going to merge as-is :)

**Pull request merged**